doc/troubleshooting.md

```diff
@@ -24,6 +24,68 @@ On Windows the `.cmd` file changes the `CWD` to the location of the script befor
 When you encounter an issue using MotoROS2, please start the debug client script and keep it running in the background while you reproduce the issue.
 Attach the log it produces and a copy of the `PANELBOX.LOG` from the robot's teach pendant to any support tickets you open on the [Issue tracker](https://github.com/yaskawa-global/motoros2/issues).
 
+## Network issues
```

Comment:
For all of these, please put each sentence on a new line. The markdown will recombobulate it into a single line. (two line breaks are required for a new line)

I don't actually know why this is a thing. But it maintains consistency throughout the doc

Reply:
This also applies to the other sections too.

Reply:
> I don't actually know why this is a thing.

because it makes the diffs much easier to read and merging changes easier.